README.md

```diff
@@ -191,9 +194,7 @@ At this point, you've seen some of the core concepts in place: declaring systems
 
 ## :book: Learn More
 
-Fides provides a variety of docs to help guide you to a successful outcome.
-
-We are committed to fostering a safe and collaborative environment, such that all interactions are governed by the [Fides Code of Conduct](https://ethyca.github.io/fides/community/code_of_conduct/).
+The Fides core team is committed to providing a variety of documentation to help get you started using Fidesctl.  As such, all interactions are governed by the [Fides Code of Conduct](https://ethyca.github.io/fides/community/code_of_conduct/).
```

Comment:
These two sentences seem totally unrelated to one another. I think the CoC deserves it's own section/heading.

Reply:
We'll need to rewrite this entire section but I think it's safe to leave in for now